### ISSUE TITLE
DO Not create an instance if startup with windows

### ISSUE THREAD

**Issue by amwihgu** [GameGodS3/DropPoint#38]
Hi, Maybe I have missed this, but it there one parameter to use to supress it from create an instance, so that I could create a shortcut into Startup folder.
Now I have to close the 1st instance manually as soon as it starts.

**Comment by papermoon1978** [comment]
No, there is no such parameter. But you could give my fork a try where this is the default behaviour.

**Comment by LisaJackson24-2** [comment]
> No, there is no such parameter. But you could give my fork a try where this is the default behaviour.

And it would be great if you replaced the shortcut combination to open in Windows with Shift + Tab.

**Comment by amwihgu** [comment]
> And could you please release your fork?

Hello, @LisaJackson24-2 
Sorry I am not an active developer on Github.
But I play a bit with Autohotkey, so here is a small hack that I've made to achieve:
1. if DropPoint.exe is running, then I send the combo key
2. if not, I run it, and it'll automatically give me the dropping panel anyway ;)
3. I changed the combo key into double click of LShift, which is only a mask level on top of the SHIFT+Capslock

```
~LShift:: 
{
   IF !(A_TickCount < LShift)
   {
      KeyWait, LShift
      LShift := A_TickCount + 200 ;1.5 seconds between releasing LAlt then pressing it again
      Return
   }
   LShift =
;    ToolTip, LShift key twice
   process, exist, DropPoint.exe
    if errorlevel
    {
        ; Send, {ShiftDown}{CapsLock}{ShiftUp}
        Send, +{CapsLock}
        SetCapsLockState, off
        Return
    } else {
        Run, "C:\Apps\DropPoint\DropPoint.exe"
        Return
    }
   Return
}
```
